src/actions/comments.js

```diff
@@ -43,12 +44,23 @@ export function appendComment (postId, comment) {
   }
 }
 
-export function createComment ({ postId, text, tagDescriptions, imageUrl }) {
+export function createComment ({ postId, text, tagDescriptions, imageUrl, userId }) {
   const params = {text, tagDescriptions, imageUrl}
+  const comment = {
+    id: uniqueId(`post${postId}_`),
+    user_id: userId,
+    thank_ids: []
+  }
+  if (text) {
+    comment.text = text
+  }
```

Comment:
sorry about the stylistic things, I've been coding on C9 still because running Atom on my computer, oddly, just doesn't go that well for me. I'm getting that sorted soon!
Also, I was going to run js-standard over these files, I'll do that next time. Just when I thought I got it right too, cause I've been trying to actually add it in to my memory too